tree_in_context/Cargo.toml

```diff
@@ -11,4 +11,4 @@ strum = "0.25"
 strum_macros = "0.25"
 main_error = "0.1.2"
 thiserror = "1.0"
-bril-rs = { git = "https://github.com/sampsyo/bril", rev = "daaff28" }
+bril-rs = { git = "https://github.com/ajpal/bril", rev = "a27a351" }
```

Comment:
Do not merge yet!
Wait for https://github.com/sampsyo/bril/pull/315 and then go back to depending on real bril